test/test.js

```diff
@@ -0,0 +1,27 @@
+const test = require('tape');
```

Comment:
I tend to use mocha and sinon as well. I'll switch to that. Used tape as it was already a dependency, but no tests...